plugin.py

```diff
@@ -26,9 +27,16 @@ def process(self, trace, data_context):
         log.info(f"processing trace {trace.get('name')}")
         # Add your plugin implementation here
 
+    # @Transformer enables the HANSKEN to execute code in the plugin
```

Comment:
`plugin_info` and `process` have no documentation.
this method has, so it's a inconsistent.

May I suggest to put a link to general transformers documentation below the method, as there is some text below the `process`-implementation?